frontend/src/components/frequentlyUsedFunctionsAndVariables.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Removes all accents from words and makes them uppercase.
+ **/
+const removeAccentsFromWords = (word) => {
```

Comment:
Ez nagybetűsít is, én simán normalize-nak hívnám.

Reply:
Átneveztem.